src/databases/OpenPMD/OpenPMDClasses/PMDField.C

```diff
@@ -863,13 +877,13 @@ PMDField::SetDataOrder(char * name,
         // Size of an elements
         size_t size = H5Tget_size (attrType);
 
-        char buffer[size+1];
+        char *buffer = new char[size+1];
 
         err = H5Aread(attrId, attrType, buffer);
         buffer[size] = '\0';
 
         this->dataOrder = buffer;
-
+		delete buffer;
```

Comment:
Replace tabs with spaces and indent properly. You can also add a blank line before the delete.